### PR TITLE
Deploy multiple datafeed (`BORA`, `KSP`, `MBX`, `WEMIX`)

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

adds multiple migration json files for new aggregators.
`BORA`-`KRW`,
`KSP`-`KRW`,
`MBX`-`KRW`,
`WEMIX`-`USDT`

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

**Review by martinkersner — APPROVED** [review]
LGTM!

File: contracts/migration/baobab/Aggregator/migration.lock
Comment (martinkersner) [inline contracts/migration/baobab/Aggregator/migration.lock:62]: @nick-bisonai Please remove this `accountList.json`

File: contracts/migration/cypress/Aggregator/migration.lock
Comment (martinkersner) [inline contracts/migration/cypress/Aggregator/migration.lock:50]: @nick-bisonai Please remove this `accountList.json`

File: contracts/migration/baobab/Aggregator/migration.lock
Comment (nick-bisonai) [reply contracts/migration/baobab/Aggregator/migration.lock]: removed it! thank you for pointing out

File: contracts/migration/cypress/Aggregator/migration.lock
Comment (nick-bisonai) [reply contracts/migration/cypress/Aggregator/migration.lock]: removed it! thank you for pointing out

**Review by martinkersner — APPROVED** [review]
LGTM! Once all test pass, let's merge it!

**Pull request merged**
